stock_picking_partner_invoice_address/models/stock_picking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+# from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    partner_invoice_id = fields.Many2one(string="Invoice Address", related="sale_id.partner_invoice_id", store=True)
+    same_invoiced_id = fields.Boolean(string="Partner = Invoice ", store=True, help="Whether or not the trading partner address is the same as the Invoice Address.")
```

Comment:
Avoid a field name ending with `_id` when it's not a many2one field. My preference: `is_identical_partners`

Reply:
fixed.I fixed this.